packages/opentelemetry-plugin-postgres/src/utils.ts

```diff
@@ -0,0 +1,155 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Span, CanonicalCode, Tracer, SpanKind } from '@opentelemetry/types';
+import { AttributeNames } from './enums';
+import {
+  PgClientExtended,
+  PgPluginQueryConfig,
+  PostgresCallback,
+  PgClientConnectionParams,
+} from './types';
+import * as pgTypes from 'pg';
+import { PostgresPlugin } from './pg';
+
+function arrayStringifyHelper<T>(arr: Array<T>): string {
```

Comment:
nit: Does this need to be generic?
```suggestion
function arrayStringifyHelper(arr: Array<unknown>): string {
```